### DISCUSSION TITLE
Also add GitHub sponsoring?!

### DISCUSSION THREAD

**Discussion by angelus2014** [NvChad/NvChad#1523] [Ideas]
Perhaps it would be a good idea to also add the GitHub 'native' sponsoring option? I already sponsor several OSS projects that way, so it's convenient to have all in one place. Also, I'm pretty sure GitHub matches all sponsorships.

---
With regards,

Angelo Machils

**Comment by siduck** [comment]
@angelus2014 i have thought about that before but github sponsors needs a stripe account right? I dont have stripe and I need PAN card to make a stripe account. I applied for the PAN card today, it'll take some weeks. So the github sponsor will be added in few weeks ig

**Comment by angelus2014** [comment]
Hello,

On 30/08/22 10:19PM, Sidhanth Rathod wrote:
>I have thought about that before but github sponsors needs a stripe 
>account right? I dont have stripe and I need PAN card to make a stripe 
>account. I applied for the PAN card today, it'll take some weeks. So 
>the github sponsor will be added in few weeks ig

As a sponsor, I don't have/need a Stripe account. Perhaps as the 
recipient you will need one?! No idea to be honest.


-- 
With regards,

Angelo Machils

**Comment by siduck** [comment]
Update, I am unable to create a stripe account because it needs a GST Number here in India and I cant get the GST tax stuff cuz I dont own any business :smile: 

if you still want to donate then you can probably use patreon https://patreon.com/siduck or kofi https://ko-fi.com/siduck

**Reply by siduck** [reply]
Edit : @angelus2014 I was able to make the account, you can give a try https://github.com/sponsors/siduck , I'll check if this works, if yes then i'll update the repo's sponsor link

**Comment by angelus2014** [comment]
Hello,

On 02/09/22 09:39PM, Sidhanth Rathod wrote:
>Edit : @angelus2014 I was able to make the account, you can give a try https://github.com/sponsors/siduck , I'll check if this works, if yes then i'll update the repo's sponsor link

It seems to work:

```
You're now sponsoring @siduck!

- Sponsorship amount: $3.00 / month
- Next billing date: September 19, 2022

Thank you for supporting @siduck's work!
```

-- 
With regards,

Angelo Machils
